public/js/controllers/traffic_metrics_controller.js

```diff
@@ -49,7 +49,8 @@ export class TrafficMetricsController {
         let node = d3.select("#AT-units-bal");
         if (data.length) {
             let { balance } = data[0];
-            node.text(balance);
+            let [ currency, amount ] = balance.split(" ");
+            node.text(`${Number(amount).toLocaleString("en-KE", { style: "currency", currency })}`);
```

Comment:
Why is this en-KE when the others are the user's default? Is there a way to pass in the default (something like navigator.language)? I think it should be user's default everywhere, for consistency.

(If not then en-KE with an explanatory comment is fine - everyone in AVF should be comfortable with this locale)

Reply:
Thought it was better because it produces the 'Ksh' sign which we are used to instead of 'KES' code but it's not a big deal. Updated thanks.